libs/event/src/lib/components/request-screening/request-screening.component.html

```diff
@@ -16,7 +16,8 @@
       <mat-spinner diameter="24" *ngIf="status === 'sending'"></mat-spinner>
```

Comment:
remove ng-container and do : 
```suggestion
      <mat-spinner *ngIf="status === 'sending'; else icon" diameter="24"></mat-spinner>
```

Reply:
and spinner is not well centered 
![image](https://user-images.githubusercontent.com/1194344/166250436-5bac7542-aa4d-4938-ba76-2f40c37e90bc.png)